### PR TITLE
Bz1433894: RHV provider snapshots support

### PR DESCRIPTION
**Pull request by cbudz**
Hey Suyog,

Here's the PR related to snapshot support for RHV provider vms. 

Let me know what you think. 

-Chris

### CODE REVIEW

File: doc-Managing_Infrastructure_and_Inventory/topics/Reverting_a_VM_snapshot.adoc [hunk 1]
```diff
@@ -15,7 +15,7 @@ To revert a virtual machine to a snapshot:
 . Select the target virtual machine, and open it to view its details.
 . Under *Properties*, click *Snapshots* to show information about the virtual machine's snapshots.
 . Select the snapshot to revert to under *Available Snapshots*.
-. Click  image:2106.png[](*Revert to selected snapshot*).
+. Click  revert_icon.png[](*Revert to selected snapshot*).
```
Comment (suyogsainkar) [inline doc-Managing_Infrastructure_and_Inventory/topics/Reverting_a_VM_snapshot.adoc]: hey @cbudz, the image isn't rendering here.

File: doc-Managing_Infrastructure_and_Inventory/topics/Creating_a_VM_snapshot.adoc [hunk 2]
```diff
@@ -6,13 +6,9 @@ To create a snapshot of a virtual machine:
...
 . Under *Properties*, click *Snapshots* to show information about the virtual machine's snapshots.
-
-+
-image:Snapshot_info.png[]
-
-. Click image:plus_circle.png[](*Create a new snapshot for this VM*).
+. Click image:plus_green.png[](*Create a new snapshot for this VM*).
 . Enter snapshot details in *Description*.
-. (Optional) Check *Snapshot VM memory* for the snapshot to preserve the virtual machine's current runtime state.
+. (Optional) Check *Snapshot VM memory* for the snapshot to preserve the virtual machine's current runtime state. This option will appear only if the VM *Power State* is image:2143.png[](*On*).
```
Comment (suyogsainkar) [inline doc-Managing_Infrastructure_and_Inventory/topics/Creating_a_VM_snapshot.adoc]: @cbudz We use Select and Clear to refer to marking a check box per IBM style guide.

**Comment by cbudz** [comment]
Hey Suyog,

I've corrected the two files and updated the preview for you to see the changes. Thanks for the feedback,

-Chris

File: doc-Managing_Infrastructure_and_Inventory/topics/Reverting_a_VM_snapshot.adoc (hunk 1 above)
Comment (cbudz) [inline doc-Managing_Infrastructure_and_Inventory/topics/Reverting_a_VM_snapshot.adoc]: I've corrected this file

File: doc-Managing_Infrastructure_and_Inventory/topics/Creating_a_VM_snapshot.adoc (hunk 2 above)
Comment (cbudz) [inline doc-Managing_Infrastructure_and_Inventory/topics/Creating_a_VM_snapshot.adoc]: Updated with standard langauge.

**Comment by suyogsainkar** [comment]
@cbudz Many thanks, Chris -- looks good. Merging to master. Cheers, Suyog

**Pull request merged**